### PR TITLE
"The image cannot be located" message appears if Save Preview when there are more than one page in Media Gallery 

### LINKED ISSUE #1254: "The image cannot be located" message appears if Save Preview when there are more than one page in Media Gallery
### Preconditions (*)
Magento 2.4-develop
Adobe-Stock-Integration 1.1-develop

1. Installed Magento with Adobe Stock Integration
2. Configured integration in Stores -> Configuration -> Advanced-> System -> Adobe Stock Integration fieldset
3. At least two pages of images on the grid
4. At least two folders in folder tree

### Steps to reproduce (*)
1. Go to **Content - Media Gallery**
2. Click **Search for Adobe Stock** 
3. Select an image and click **Save Preview**, **Confirm**
4. Switch to another folder from the folder tree 

### Expected result (*)
Change Sorting to Newest first

### Actual result (*)
_The image cannot be located
We cannot find this image in the media gallery._ error message appears 
the image cannot be located_2204

im_cannot_be loc_two pages_2204

### Please check the following scenario while testing this issue as well
0. Disable Enhanced Media Gallery
1. Save Preview image from Adobe Stock Integration
2. Remove the saved image from the filesystem
3. Reopen Adobe Stock Integration preview for saved image
4. Click "Open in Media Gallery" button

Expected result:
"The image cannot be located" message is shown

### PR DESCRIPTION
**Pull request by engcom-Charlie**
### Description (*)

Closes #1254

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1254: "The image cannot be located" message appears if Save Preview when there are more than one page in Media Gallery 
2. ...

Depends on https://github.com/magento/adobe-stock-integration/pull/1275

### CODE REVIEW

**Comment by engcom-Charlie** [comment]
It needs to recheck after merge https://github.com/magento/adobe-stock-integration/pull/1230

File: MediaGalleryUi/view/adminhtml/web/js/image-uploader.js [hunk 1]
```diff
@@ -112,6 +114,9 @@ define([
 
                         return;
                     }
+                    this.sortBy().selectDefaultOption();
```
Comment (Nazar65) [inline MediaGalleryUi/view/adminhtml/web/js/image-uploader.js]: In this case if we upload for example 10 images, this operation will  be executed 10 time's

**Review by Nazar65 — CHANGES_REQUESTED** [review]

File: MediaGalleryUi/view/adminhtml/web/js/image-uploader.js (hunk 1 above)
Comment (Nazar65) [reply MediaGalleryUi/view/adminhtml/web/js/image-uploader.js]: I would recommend to move this operation to stop method to run this on last uploaded image

**Review by lenaorobei — CHANGES_REQUESTED** [review]
Please adjust code with changes from https://github.com/magento/adobe-stock-integration/pull/1275

**Review by sivaschenko — DISMISSED** [review]

**Review by chalov-anton — CHANGES_REQUESTED** [review]
@engcom-Charlie Please, add behaviour when a selected image from Media Gallery is not deselected after Uploading a new image 
uploade_deselect

**Review by sivaschenko — APPROVED** [review]

**Review by chalov-anton — COMMENTED** [review]
:heavy_check_mark: QA Passed

**Pull request merged**